ckc/stripe/utils/subscriptions.py

```diff
@@ -0,0 +1,39 @@
+import stripe
```

Comment:
I actually think we don't need this `utils/` subdir? `payments.py`, and `subscriptions.py` could live right in `ckc.stripe`